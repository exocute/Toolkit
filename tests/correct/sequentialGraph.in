Graph Sport.Lisboa.e.Benfica
	Import Penalty.por.assinalar
	Export dois.um
	Activity goloSLB1 Eduardo.Salvio:24:Rafael.Silva
		Import Penalty.por.assinalar
		Export Bola.ao.poste
	Activity goloSLB2 Raul.Jimenez:47:Nelson.Semedo
		Import Bola.ao.poste
		Export Lindelof.a.dormir
	Activity goloSCP1 Bas.Dost:69:Joel.Campbell
		Import Lindelof.a.dormir
		Export esperanca.no.empate
	Activity fim Bas.Dost:69:Joel.Campbell
		Import esperanca.no.empate
		Export dois.um
		
	connection [goloSLB1;goloSLB2][goloSLB2;goloSCP1][goloSCP1;fim]