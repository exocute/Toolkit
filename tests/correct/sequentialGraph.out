Name: Sport.Lisboa.e.Benfica
ImportName: Penalty.por.assinalar
ExportName: dois.um
Id: fim, Name: Bas.Dost, Import: esperanca.no.empate, Export: dois.um, Parameters: 69,Joel.Campbell
Id: goloSCP1, Name: Bas.Dost, Import: Lindelof.a.dormir, Export: esperanca.no.empate, Parameters: 69,Joel.Campbell
Id: goloSLB1, Name: Eduardo.Salvio, Import: Penalty.por.assinalar, Export: Bola.ao.poste, Parameters: 24,Rafael.Silva
Id: goloSLB2, Name: Raul.Jimenez, Import: Bola.ao.poste, Export: Lindelof.a.dormir, Parameters: 47,Nelson.Semedo
goloSCP1:fim
goloSLB1:goloSLB2
goloSLB2:goloSCP1